modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -901,12 +901,26 @@ definition {
 	}
 
 	@description = "Verify that is possible to delete translations"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanDeleteTranslations {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanDeleteTranslations";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
-		// TODO CanDeleteTranslations pending implementation
+		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(translationOptionSelected = "en-us");
+
+		ProcessBuilderKaleoDesignerReact.viewTranslationLanguagePresentOnDropdownMenu(translationOption = "fr_FR");
+
```

Comment:
The assert that you could make here, should be on manage translations component/modal, if the language that will be deleted it's present there. This works too? Yes! But it's not the goal if this testcase.